pubspec.yaml

```diff
@@ -11,7 +11,7 @@ authors:
   - Evan Weible <evan.weible@workiva.com>
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ^2.2.1
```

Comment:
We can't use this syntax here because older versions of Dart SDK don't recognize it:
```
Missing requirements:
* ^ version constraints aren't allowed for SDK constraints since older versions of pub don't support them.
  Expand it manually instead:

  environment:
    sdk: ">=2.2.1 <3.0.0"
```

Reply:
Huh didn't know this, we'll have to update the d1down codemod.

Reply:
https://github.com/Workiva/dart2_upgrade_helper/pull/7